dipy/tracking/direction_getter.pxd

```diff
@@ -1,14 +1,28 @@
 
-cimport numpy as np
+from .direction_getter cimport DirectionGetter
```

Comment:
I do not think this import is needed. It is strange here

Reply:
yes, it not needed. removed.